client/my-sites/themes/collections/collection-definitions.tsx

```diff
@@ -1,6 +1,20 @@
 import { translate } from 'i18n-calypso';
 
 export const THEME_COLLECTIONS = {
+	recommended: {
+		query: {
+			collection: 'recommended',
+			filter: '',
+			number: 10,
```

Comment:
Not sure why, but this is not limited to 10: it returns all 129 curated themes, starting with Peirao and ending with Natural Block. It's quite obvious if you inspect the DOM.

Reply:
🤔 what is is that you're seeing? It's not looking obvious to me

<img width="957" alt="Screenshot 2023-10-18 at 16 31 07" src="https://github.com/Automattic/wp-calypso/assets/93301/d3eea6d6-0c65-4709-aafd-2c78167f4463">

The API query is https://public-api.wordpress.com/rest/v1.2/themes?http_envelope=1&collection=recommended&filter=&number=10&page=1&search=&tier=&include_blankcanvas_theme= which returns ten themes for me.

Reply:
It seems to be a caching issue, the API is indeed returning 10 results. I'll keep an eye on this and maybe trim the results in the carousel component (which fetches the data).

Reply:
Not sure what happened! I was testing on a fresh incognito, so it would be weird if it had cached stuff. 🤔 
Anyway, it's a non-blocking issue that we can address when (if!) we see it again.